extensions/tekton/tests/src/test/java/io/fabric8/tekton/test/crud/V1alpha1PipelineCrudTest.java

```diff
@@ -20,7 +20,7 @@
 import io.fabric8.tekton.pipeline.v1alpha1.Pipeline;
 import io.fabric8.tekton.pipeline.v1alpha1.PipelineBuilder;
 import io.fabric8.tekton.pipeline.v1alpha1.PipelineList;
-import io.fabric8.tekton.v1alpha1.internal.pipeline.pkg.apis.pipeline.v1alpha2.Param;
+import io.fabric8.tekton.v1alpha1.internal.pipeline.pkg.apis.pipeline.v1beta1.Param;
```

Comment:
The test suite is named `V1alpha1PipelineCrudTest`

Maybe we should create a `V1beta1PipelineCrudTest` suite with v1beta1 specific tests.

Reply:
Ok, sorry, missed the first part of the import. I suppose something has changed in the base v1alpha1 api (but this change is weird for me nonetheless)

Reply:
that's caused by the update of the v1alpha1 dependency